content/authors/administrator-robin-carnahan/_index.md

```diff
@@ -5,31 +5,10 @@
-# List your pronoun(s) if you want them displayed alongside your name. Leave it blank and we'll use just your name.
-# See https://uwm.edu/lgbtrc/support/gender-pronouns/ for a list of pronouns
-# Examples: they/them, she/her, or he/him
-pronoun: 
-
 # slug — the specific user-id for an author.
 slug: administrator-robin-carnahan
-
-# if you include an email address, it will be displayed on your profile page
-email: 
-
-# Bio — keep it under 50 words
-bio: "Robin Carnahan currently serves as the Administrator of the U.S. General Services Administration (GSA).<br /><br />As Administrator, she is working to empower GSA career professionals and build on the agency’s efforts to deliver the best value in real estate, acquisition, and technology services to the government and American people.<br /><br />Prior to joining GSA, Carnahan served in executive and leadership roles in business, academia and government, including as the Secretary of State of Missouri (2005-2013), and founded and led the State and Local Government Practice at 18F, a tech consultancy, inside GSA (2016–2020). She’s a nationally recognized government technology leader and in 2017 was named one of the federal government’s “Top Women in Tech.” Most recently, Carnahan was a Fellow at Georgetown University’s Beeck Center where she co-founded the State Software Collaborative.<br /><br />As Secretary of State, Carnahan also served as the state’s Chief Election Official and State Securities Regulator and was responsible for providing in-person and online services to hundreds of thousands of customers. An essential part of her job was leading the office’s technology modernization efforts across 7 operating divisions. She frequently speaks, writes and testifies about government innovation through smarter use of technology.<br /><br />While previously at GSA, Carnahan helped federal, state, and local government agencies improve customer facing digital services and cut costs. She focused on training and empowering non-technical executives on ways to reduce risk and deliver better results for the public by more effectively budgeting, procuring, implementing, and overseeing digital modernization projects.<br /><br />Carnahan holds a bachelor’s degree in Economics from William Jewell College and a J.D. from the University of Virginia School of Law."
-
-# Where can people learn more about your agency or program? Provide a full URL [e.g. 'https://www.example.gov/']
-bio_url: "https://www.gsa.gov/about-us/organization/leadership-directory/administrator"
-
-# Agency Full Name [e.g. U.S. General Services Administration]
-agency_full_name: "U.S. General Services Administration"
-
 # Agency Acronym [e.g., GSA]
 agency: "GSA"
-
 # Tell us where you live and work [e.g. 'New York City' or 'Portland, OR']
 location: ""
```

Comment:
```suggestion
```
All of this metadata should be removed if we're not going to ever include location again